krping.c

```diff
@@ -1924,6 +1926,10 @@ static void krping_run_client(struct krping_cb *cb)
 	struct ib_recv_wr *bad_wr;
 	int ret;
 
+	/* set type of service, if any */
+	if (cb->tos != 0)
```

Comment:
Is 0 not a valid service type?

Reply:
0 is the default TOS, no need to set it from what I can see.

Reply:
ok then.  The PR looks good. Thanks!